src/components/SelectedMeeting/SelectedMeeting.js

```diff
@@ -41,7 +41,6 @@ const edit = (e) =>  {
     //map this out, research stack
     <div>
        <MeetingNavBar prop={'details'}/>
-     <div style={{marginTop: '86px'}}>
```

Comment:
div to box, set max width to the full screen (should probably be width but idk right now)